native_client/Makefile

```diff
@@ -56,15 +56,15 @@ bindings: setup.cfg
 run: deepspeech
 	${META_LD_LIBRARY_PATH}=${TFDIR}/bazel-bin/tensorflow:${TFDIR}/bazel-bin/native_client:${${META_LD_LIBRARY_PATH}} ./deepspeech ${ARGS}
 
+debug: deepspeech
+	${META_LD_LIBRARY_PATH}=${TFDIR}/bazel-bin/tensorflow:${TFDIR}/bazel-bin/native_client:${${META_LD_LIBRARY_PATH}} gdb --args ./deepspeech ${ARGS}
+
 install:
```

Comment:
The install target only installs the libraries right now - It'd make sense for it to install the binary too though, I'll change that.